pkg/types/validation/installconfig.go

```diff
@@ -304,3 +307,18 @@ func validateNamedRepository(r string) error {
 	}
 	return nil
 }
+
+var (
+	validPublishingStrategies = map[types.PublishingStrategy]bool{
+		types.ExternalPublishingStrategy: true,
```

Comment:
nit: in other parts of this repo, we use `map[...]struct{}` for existence-checking sets.  Dunno if we care about consistency; I'm fine either way.

Reply:
fixed with https://github.com/openshift/installer/compare/586800a7998182fd87bc00700adf24a49df216be...83adc04d6cf70ffc2f45bbe6a5e8d1d06173832d